cob_bms_driver/src/power_state_aggregator.py

```diff
@@ -33,12 +33,14 @@ def __init__(self):
         self.current = None
         self.last_currents = []
         self.last_update = rospy.Time(0)
+        self.connected = False
         self.charging = False
         self.remaining_capacity = None
         self.full_charge_capacity = None
         self.temperature = None
         rospy.Subscriber("voltage", Float64, self.voltage_cb)
         rospy.Subscriber("current", Float64, self.current_cb)
+        rospy.Subscriber("connected", Float64, self.current_cb)
```

Comment:
this subscriber needs to be removed

Reply:
fixed in https://github.com/ipa320/cob_driver/pull/434/commits/85c2c33bafca5171e70d9a235b64ebe77ab6c6b7